dbms/src/Flash/Planner/PhysicalPlanNode.h

```diff
@@ -75,6 +78,8 @@ class PhysicalPlanNode : public std::enable_shared_from_this<PhysicalPlanNode>
         Context & /*context*/,
         PipelineExecutorStatus & /*exec_status*/);
 
+    EventPtr sinkFinalize(PipelineExecutorStatus & exec_status);
```

Comment:
The two are not related.
`finalize` is used to process expression action
`sinkFinalize` is to support two-stage logic, like local/global hash join build and local/final agg spill(non-final-grained).
Perhaps `sinkFinalize` needs to be renamed to avoid confusion?

Reply:
renamed to `sinkComplete`

Reply:
> The two are not related. `finalize` is used to process expression action `sinkFinalize` is to support two-stage logic, like local/global hash join build and local/final agg spill(non-final-grained). Perhaps `sinkFinalize` needs to be renamed to avoid confusion?

okk